core/web-assets/src/main/assets/js/apps/onms-resources/index.js

```diff
@@ -105,7 +106,7 @@ angular.module('onms-resources', [
       $scope.loaded = true;
       $scope.hasResources = data.children.resource.length > 0;
       var reduced = _.map(data.children.resource, function(obj) {
-	var resource = {
+	    var resource = {
```

Comment:
Looks like the indenting of the object's fields needs to be updated since this is being updated.